pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -246,14 +285,14 @@ stdenv.mkDerivation ({
 
   postPatch = optionalString jailbreak ''
     echo "Run jailbreak-cabal to lift version restrictions on build inputs."
-    ${jailbreak-cabal}/bin/jailbreak-cabal ${pname}.cabal
+    ${stdenv.lib.getBin jailbreak-cabal}/bin/jailbreak-cabal ${pname}.cabal
```

Comment:
It's not needed to call the `getBin` here. The rule of multiple outputs is that `${foo}/bin/bar` will always, always do the right thing.